common/src/com/thoughtworks/go/util/validators/JettyWorkDirValidator.java

```diff
@@ -29,6 +29,7 @@
     public Validation validate(Validation val) {
         if (SystemEnvironment.getProperty("jetty.home", "").equals("")) {
             new SystemEnvironment().setProperty("jetty.home", new SystemEnvironment().getPropertyImpl("user.dir"));
+            new SystemEnvironment().setProperty("jetty.base", new SystemEnvironment().getPropertyImpl("user.dir"));
```

Comment:
Is there a possibility that jetty.home is not empty, but jetty.base is? In that case, jetty.base will be left empty.